api/compat.py

```diff
@@ -76,12 +108,52 @@ async def contributors(request: Request) -> JSONResponse:
         - HTTPException: If there is an error retrieving the releases.
     """
 
-    data: dict[str, list] = {
-        "repositories": await github_backend.compat_get_contributors(
-            repositories=[
-                GithubRepository(owner=owner, name=repo) for repo in compat_repositories
-            ]
+    return json(await generate_contributors_data(), status=200)
+
+
+@cli.helper(name="tools")
```

Comment:
May be out of scope for this PR but with the routes manager and patches this route is obsolete